Mathlib/SetTheory/Cardinal/Basic.lean

```diff
@@ -2027,6 +2032,17 @@ theorem mk_preimage_of_injective_of_subset_range (f : α → β) (s : Set β) (h
     (h2 : s ⊆ range f) : #(f ⁻¹' s) = #s := by
   convert mk_preimage_of_injective_of_subset_range_lift.{u, u} f s h h2 using 1 <;> rw [lift_id]
 
+@[simp]
+theorem mk_preimage_equiv_lift {β : Type v} (f : α ≃ β) (s : Set β) :
+    lift.{v} #(f ⁻¹' s) = lift.{u} #s := by
```

Comment:
Do we already have the `image` version of this too?

Reply:
I've added these to the PR - note that the `image` version can be stated with just an embedding instead of an equiv.

Reply:
Yup, nice. I wonder if you can golf `mk_preimage_equiv_lift` using the image one...